src/components/Feature.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Feature = ({
+  icon_style = "",
+  src = "",
+  alt = "",
+  text_container_style = "",
```

Comment:
What is this? Why are you using a snake case? On other places you used camel case, and here?